src/Dofs/sparsity_pattern.jl

```diff
@@ -1,55 +1,59 @@
 """
-    create_sparsity_pattern(dh::DofHandler; coupling)
+    create_sparsity_pattern(dh::DofHandler; coupling, topology::Union{Nothing, AbstractTopology}, elements_coupling)
```

Comment:
Can you clarify how one opt-in to getting cross element couplings now. Do you need to pass `elements_coupling` *and* topology?